### ISSUE TITLE
マイクを選択可能にする

### ISSUE THREAD

**Issue by 1heisuzuki** [1heisuzuki/speech-to-text-webcam-overlay#37]
カメラの選択部分を参考にする
0252c3f40bab91a4a08c679ac05db9d4b9888ae8

**Comment by sksthrs** [comment]
カメラ選択の時に、私もマイクも選べたらいいなと思ったのですが、軽く調べた範囲では、Web Speech APIには音声ソースを選ぶ機能がありませんでした。何か方法があればよいのですが。

**Comment by 1heisuzuki** [comment]
なるほどー！ありがとうございます。

既定のマイクをJSから編集出来たらいけるかな？と思ったんですが，どうやらそれはできなさそうですね。
https://support.google.com/chrome/thread/50735502?hl=ja

**Comment by 1heisuzuki** [comment]
このメディア許可用ポップアップの挙動も違うのでヒントになりそうですね。普段そこまでWebアプリ等書いているわけではなく，まったくわかりませんが…

本Webページのとき
<img width="347" alt="SS 2020-06-03 at 19 31 13" src="https://user-images.githubusercontent.com/14171207/83626727-d4314200-a5d0-11ea-9bb7-be064c5be6f9.png">


Google Meetのとき
<img width="347" alt="SS 2020-06-03 at 19 31 13" src="https://user-images.githubusercontent.com/14171207/83626795-eca15c80-a5d0-11ea-9fd4-fd1ef89eb084.png">

**Comment by 1heisuzuki** [comment]
あとはChromiumのコードを漁るとかですかね…
https://www.chromium.org/Home

**Comment by sksthrs** [comment]
そうですね……。getUserMediaでとったマイクをSpeech Recognition APIに渡す形なら対応できるのですが、これはむしろChrome側に出すべき案件かもしれませんね。
